prisma/schema.prisma

```diff
@@ -77,6 +77,8 @@ model User {
   streak                    Int?
   gunStreak                 Int?
   horseStreak               Int?
+  hasSendWallet             Boolean              @default(false)
+  hasRecvWallet             Boolean              @default(false)
```

Comment:
what's the point of these columns if the resolvers don't use them?

Reply:
Oh, that was my initial idea but something must have changed my mind and then I forgot to clean this up :grimacing: 

I think I realized that `hasSendWallet` won't work without device sync but it also doesn't work without with the resolver approach, so not sure what I was thinking.

I'll try to set these columns and then use them in the resolver.

Reply:
These columns are now set via triggers, see 019cdace.

Reply:
As discussed, I can simply create a job that then checks if receive or send is set and then update `hasSendWallet`/`hasRecvWallet` from there.

Reply:
Done in a2f61cb9